allauth/socialaccount/providers/google/tests.py

```diff
@@ -204,3 +204,15 @@ def test_social_email_verification_optional(self):
         self.assertEqual(len(mail.outbox), 1)
         self.login(self.get_mocked_response(verified_email=False))
         self.assertEqual(len(mail.outbox), 1)
+
+    def test_callback_android(self):
```

Comment:
At the very least, I get an access token and a refresh token when running on my machine. I'm currently operating under the assumption that once I get those tokens I'm home free. Is there anything there I'm missing?